Source/WebCore/workers/service/server/SWServerWorker.cpp

```diff
@@ -208,6 +208,9 @@ void SWServerWorker::didFinishInstall(const std::optional<ServiceWorkerJobDataId
     RELEASE_ASSERT_WITH_MESSAGE(state == ServiceWorkerState::Installing, "State is %hhu", static_cast<uint8_t>(state));
     if (RefPtr server = m_server.get())
         server->didFinishInstall(jobDataIdentifier, *this, wasSuccessful);
+    ASSERT(m_server && this->state() == ServiceWorkerState::Installing);
```

Comment:
I think having m_server in the ASSERT is wrong and the assert should be:
    ASSERT(this->state() == ServiceWorkerState::Installing);